src/openforms/conf/base.py

```diff
@@ -850,6 +850,7 @@
     },
     "ENUM_NAME_OVERRIDES": {
         "IncompleteSubmissionsRemovalMethodEnum": "openforms.data_removal.constants.RemovalMethods",
+        "AvailableLanguagesEnum": "django.conf.settings.LANGUAGES",
```

Comment:
I saw the warning in the failing CI build, but I'm curious if this is still needed when a different serializer + custom field is used for the language code choices

Reply:
That's based on the value of the enum, not the name. It warns. But in this case it's valid, they are after all the same Enum i.e. the available languages.